frontend/public/components/monitoring/alerting.tsx

```diff
@@ -1741,7 +1801,7 @@ const RuleTableRow: React.FC<RowProps<Rule>> = ({ obj }) => {
         {_.isEmpty(obj.alerts) ? '-' : <StateCounts alerts={obj.alerts} />}
       </td>
       <td className={tableRuleClasses[3]} title={title}>
-        {alertingRuleSource(obj) === AlertSource.User ? t('public~User') : t('public~Platform')}
+        {getSourceKey(_.startCase(alertingRuleSource(obj)))}
```

Comment:
Sorry, I didn't explain this well. There's no need to refactor away existing uses of `getSourceKey` with this PR, but I think it would be better to not add an additional call to `getSourceKey` here. In other words, I think this line is better left unchanged.